Makefile

```diff
@@ -34,13 +34,14 @@ BOTTLEROCKET_SDK_ARCH = $(UNAME_ARCH)
 
 BUILDER_IMAGE = public.ecr.aws/bottlerocket/bottlerocket-sdk-$(BOTTLEROCKET_SDK_ARCH):$(BOTTLEROCKET_SDK_VERSION)
 
+export CARGO_ENV_VARS = CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse
```

Comment:
Is there a reason not to just `export CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse`?

Reply:
I think this is due to how env passing in Makefile works. It doesn't behave like a bash script.